classes/application.py

```diff
@@ -96,57 +96,56 @@ def get_file(self, file):
         print("Getting data from source:", file)
         filename = os.path.join(self.source_folder, self.filenames[file])
         data = get_data(filename)
-        for key in data:
-            if key == "Sheet1":
-                data = data[key]
-                row_index = 1
-                for row in data:
-                    if row_index > 1:
-                        if file == "chief":
-                            code = self.check(row, 0).strip()
-                            direction = self.check(row, 1)
-                            level = self.check(row, 2)
-                            description = self.check(row, 3)
-                            guidance = self.check(row, 4)
-                            status_codes_cds = ""
-                        else:
-                            code = self.check(row, 0).strip()
-                            direction = self.check(row, 1)
-                            description = self.check(row, 2)
-                            guidance = self.check(row, 3)
-                            status_codes_cds = self.check(row, 4)
-                            level = ""
-
-                        if code != "":
-                            document_code = DocumentCode(
-                                file,
-                                code,
-                                direction,
-                                level,
-                                description,
-                                guidance,
-                                status_codes_cds,
-                            )
-
-                            if file == "cds_union":
-                                self.document_codes_union[
-                                    document_code.code
-                                ] = document_code.as_dict()
-                            elif file == "cds_national":
-                                self.document_codes_national[
-                                    document_code.code
-                                ] = document_code.as_dict()
-                            elif file == "chief":
-                                self.document_codes_chief[
-                                    document_code.code
-                                ] = document_code.as_dict()
-
-                            if code not in self.codes_on_govuk:
-                                self.codes_on_govuk.append(code)
-                    row_index += 1
+
+        for row in next(iter(data.values()))[1:]:
```

Comment:
I'm reading this as:

1. Convert all of the values to an iterator
2. Pull the first one from the iterator with next
3. Pull all rows except the header row

Makes sense!

Reply:
Would it be more friendly to extract

```python
  rows = next(iter(data.values()))[1:]
```

Reply:
Yeah, exactly!